GVFS/GVFS.Service/RepoRegistry.cs

```diff
@@ -178,19 +181,27 @@ public void AutoMountRepos(string userId, int sessionId)
                     return;
                 }
 
-                this.SendNotification(sessionId, "GVFS AutoMount", "Attempting to mount {0} GVFS repo(s)", activeRepos.Count);
+                this.SendNotification(
+                    sessionId,
+                    NamedPipeMessages.Notification.Request.Identifier.AutomountStart,

```

Comment:
How should we thinking about interacting with users on macOS via notifications? Do they expect these to be fired only for special situations, or do they expect chatty notifications? I worry that notifying that we're about to mount  X repos and then firing X success messages is too much. We could
1. Only fire failure to mount
2. Continue firing start, but only fire one notification summarizing what mounted successfully.

Thoughts?

Reply:
The more I think about it, the more I like option 2. It's also a nice clue that you can start working. Thoughts?

Reply:
yes, that makes sense. Let me take a look at the options to minimize auto-mount notifications count...

Reply:
As discussed - notifying only on auto-mount failures.